spec/syntax.md

```diff
@@ -691,6 +697,39 @@ _Markup_ _placeholders_ can appear in any order without making the _message_ inv
 However, specifications or implementations defining _markup_ might impose requirements
 on the pairing, ordering, or contents of _markup_ during _formatting_.
 
+## Attributes
+
+An **_<dfn>attribute</dfn>_** is a key with an optional value that
+is attached to the _expression_ or _markup_ as a whole.
+_Attributes_ are reserved for use by future specification versions.
```

Comment:
I would put this sentence first and perhaps in bold in its own paragraph? I would make it more emphatic because we give examples later and we don't want to give the wrong impression.

```suggestion
**_Attributes_ are reserved for standardization by future versions of this specification.**
Examples in this section are meant to be illustrative and
might not match future requirements or usage.
```